co2eq/food/ingredients.js

```diff
@@ -9,20 +10,22 @@ import {
   getChecksumOfFootprints,
 } from '../purchase';
 
-export const modelName = 'model name';
+export const modelName = 'meal-from-ingredients';
 export const modelVersion = `1_${getChecksumOfFootprints()}`;
 export const explanation = {
   text: 'The calculations take into consideration emissions of greenhouse gases across the whole lifecycle of the ingredients.',
   links: [
-    { label: 'Nature (2017)', href: 'https://www.nature.com/articles/s41598-017-06466-8' },
     { label: 'Tomorrow footprint database', href: 'https://github.com/tmrowco/northapp-contrib/blob/master/co2eq/purchase/footprints.yml' },
   ],
 };
 
 export const modelCanRunVersion = 1;
 export function modelCanRun(activity) {
-  const { lineItems } = activity;
-  if (lineItems && lineItems.length) {
+  const {
+    mealType,
+    lineItems,
+  } = activity;
+  if (mealType || (lineItems && lineItems.length)) {
```

Comment:
You should check the activityType === ACTIVITY_TYPE_MEAL) and the presence of ingredients)